raiden/accounts.py

```diff
@@ -10,10 +10,12 @@
 def find_datadir():
     home = os.path.expanduser("~")
     datadir = None
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('FreeBSD'):
```

Comment:
I know but this would return `True` also on MacoSX according to the documentation, which is not what we want.